### PR TITLE
#35 Implement ToolCommandlet for Terraform

### PR DESCRIPTION
**Pull request by CREITZ25**
#35 Implement ToolCommandlet for Terraform

### CODE REVIEW

File: cli/src/main/java/com/devonfw/tools/ide/tool/terraform/Terraform.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class Terraform extends ToolCommandlet {
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public Terraform(IdeContext context) {
+
+    super(context, "terraform", Set.of(TAG_BUILD));
```
[the remark this answers is not in the document]
Comment (CREITZ25) [reply cli/src/main/java/com/devonfw/tools/ide/tool/terraform/Terraform.java:20]: adapted

File: .gitignore [hunk 2]
```diff
@@ -16,3 +16,4 @@ generated/
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+idea.properties
```
[the remark this answers is not in the document]
Comment (CREITZ25) [reply .gitignore]: I changed my configuration, so this change in .gitignore is not necessary.

**Review by CREITZ25 — COMMENTED** [review]
All adapted

**Review by hohwille — APPROVED** [review]
@CREITZ25 great. Thanks for the update 👍 Now ready for merge.

**Pull request merged**